test/localize-behavior/localize-behavior.js

```
@@ -25,4 +25,30 @@ describe('localize behavior', function() {
 		expect(component.localize('allCourses')).to.equal('All Courses');
 	});
 
+	describe('localize mappings', function() {
+		it('should have translation for every english term', function() {
```

I'm roughly on the same page as you, I did add a separate test that checks that the value of what each translation is (currently commented out till we have proper translations).  My goal is I want it hard for someone to skip something that is required for our application to work in all langs. 

The reason I don't like useKeyIfMissing, is there's really no alert telling us "hey you didn't translate this thing", in general my preference is that if it's required a unit test should fail on it missing.